api/v1/errorHandler.js

```diff
@@ -122,7 +122,7 @@ module.exports = function errorHandler(err, req, res, next) {
       if (/Route defined in Swagger specification.*/.test(err.message)) {
         err.status = constants.httpStatus.NOT_ALLOWED;
       } else if (err.name === 'SequelizeUniqueConstraintError') {
-        err.status = constants.httpStatus.FORBIDDEN;
```

Comment:
This commit changes the status for SequelizeUniqueConstraintErrors to 400 so it matches DuplicateResourceError. This is necessary so the same tests will pass when the cache is enabled or disabled.